ccd-definition/CaseEvent/Submitted.json

```diff
@@ -272,5 +272,19 @@
     "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/placement/about-to-start",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/placement/about-to-submit",
     "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/placement/submitted"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "handleSupplementaryEvidence",
+    "Name": "Handle supplementary evidence",
+    "Description": "Handle supplementary evidence",
+    "DisplayOrder": 18,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
```

Comment:
i would use dedicated file for this, please coordinate what filename it is with Piotr, as he is using something similar
see
https://github.com/hmcts/fpl-ccd-configuration/pull/826/files#diff-9ad485151202c361c067deb09d40a9d9